src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -218,5 +218,28 @@ bool BulletPhysicsManager::contactTest(const int physObjectID) {
       ->contactTest();
 }
 
+btCollisionWorld::AllHitsRayResultCallback BulletPhysicsManager::castRay(
```

Comment:
We are planning to get Bullet collision object ray-casting into master as a stand-alone feature in the next week or so. 👍